owlify/src/test/java/org/archcnl/owlify/core/AbstractOwlifyComponentTest.java

```diff
@@ -0,0 +1,51 @@
+package org.archcnl.owlify.core;
+
+import static org.junit.Assert.*;
+
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.Map;
+
+import org.apache.jena.rdf.model.Model;
+import org.junit.Test;
+
+public class AbstractOwlifyComponentTest extends AbstractOwlifyComponent {
+	
+	// AbstractOwlifyComponent is the abstract base class for Input Parsers
+	@Test
+	public void givenPath_whenAddedToAbstractOwlifyComponent_thenPathIsProperlyAdded() {
+		// given
+		String testPath = "c:/dir1/dir2/dir3";
+		String testPath2 = "c:/dir1/dir2/dir3/dir4";
```

Comment:
It's better not to use Windows-specific paths for this. Instead, you could use a directory in the user's home directory. The home directory can be addressed in an OS unspecific way with:
`System.getProperty("user.home");`